app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person in people:
+        person_list.append(Person(name=person["name"], age=person["age"]))
+
+    for person in people:
+        current_person = Person.people[person["name"]]
```

Comment:
Accessing `Person.people[person['name']]` assumes that the person already exists in the dictionary. Ensure that all names in the input list are unique and have been added to the dictionary.